vllm/model_executor/layers/quantization/compressed_tensors/schemes/compressed_tensors_w8a8_statictensor.py

```diff
@@ -0,0 +1,126 @@
+from typing import Callable, List, Tuple, Union
+
+import torch
+from torch.nn import Parameter
+
+# TODO (varun) : Unify ops and custom ops
```

Comment:
This should not be left as a TODO and instead be done before the PR is merged -- it is a very small amount of work

Reply:
Yes, definitely. This skipped my radar. This is fixed now. Thanks for catching it.